Don't read CHANGELOG.md: it is not included in the PyPI sources
Author: Samuel Moors (Vrije Universiteit Brussel)
diff -ur sacrebleu-2.4.1/setup.py sacrebleu-2.4.1.new/setup.py
--- sacrebleu-2.4.1/setup.py	2024-03-12 15:39:06.000000000 +0100
+++ sacrebleu-2.4.1.new/setup.py	2024-04-03 11:20:48.364104000 +0200
@@ -77,14 +77,6 @@
     with open('README.md') as f:
         long_description = f.read()
 
-    with open('CHANGELOG.md') as f:
-        release_notes = f.read()
-
-    # Plug release notes into the long description
-    long_description = long_description.replace(
-        '# Release Notes\n\nPlease see [CHANGELOG.md](CHANGELOG.md) for release notes.',
-        release_notes)
-
     return long_description
 
 
